presto-main/src/main/java/com/facebook/presto/operator/DriverContext.java

```diff
@@ -235,6 +248,17 @@ public void freeMemory(long bytes)
         memoryReservation.getAndAdd(-bytes);
     }
 
+    public void freeRevocableMemory(long bytes)
+    {
+        if (bytes == 0) {
```

Comment:
I copied/pasted and modified `freeMemory` method. I think @yuananf has introduced this earlier this year.